doc/release-notes.md

```diff
@@ -1,65 +1,50 @@
 # Changes to `ga4gh/schemas` `master` branch since version `v0.5.1` (Oct 2, 2014)
 
-### Pervasive changes
+## Pervasive changes
 
 Renamed all protocols and records to remove the "`GA`" prefix from
 their names.  (Hence `GASearchVariantSetsRequest` is now
-`SearchVariantSetsRequest`, etc.)  The exception is `GAException`,
-which is unchanged.
+`SearchVariantSetsRequest`, etc.)  
+The exception is `GAException`, which is unchanged.
 
 There is no longer one unitary namespace.  Objects now reside in
 "`org.ga4gh.models`", methods in "`org.ga4gh.methods`".
 
-Updated the version number to `0.6.0-SNAPSHOT`.
+Updated the version number to `0.6.a`.
 
-### Added
 
-Datasets are now used throughout the API.
+## Changes visible to API clients
 
-#### Methods:
+### Additions
 
-* `getDataset`
-* `searchDatasets`
-* `getReadGroupSet`
-* `getReadGroup`
-* `getVariant`
-* `getVariantSet`
-* `getCallSet`
+`Dataset` is now utilized throughout the API.
 
-#### Records:
+#### New methods, HTTP endpoints:
 
-* `ExternalIdentifier`
-* `Experiment`
-* `Dataset`
-* `ReadStats`
-* `Fragment` _(apparently unused)_
+* `getDataset` via GET `datasets/<id>`
+* `searchDatasets` via POST `/datasets/search`
+* `getReadGroupSet` via  GET `/readgroupsets/<id>`
+* `getReadGroup` via GET `/readgroups/<id>`
+* `getVariant` via GET `/variants/<id>`
+* `getVariantSet` via GET `/variantsets/<id>`
+* `getCallSet` via GET `/callsets/<id>`
 
-#### Enum:
+#### New and modified entities:
 
-* `Strand`
-
-### Removed
-
-* `src/main/resources/avro/beacon.avdl`
-* Files designated "Work In Progress" ("`wip`"):
-    * `src/main/resources/avro/wip/metadata.avdl`
-    * `src/main/resources/avro/wip/metadatamethods.avdl`
-    * `src/main/resources/avro/wip/variationReference.avdl`
-
-### Changed
-
-Moved `GAException` to `methods.avdl`.
+* New record type `ExternalIdentifier` introduced (not currently used - reserved for sequence annotations?)
```

Comment:
I would drop the editorial comment from the release notes, i.e. remove " - reserved for sequence annotations?" and resolve that separately.